### ISSUE TITLE
Taxonomic rank/values get removed at dataset submission

### ISSUE THREAD

**Issue by csjx** [NCEAS/metacatui#1574]
**Description**
In the metadata editor, adding multiple taxa rank name and value pairs works well. The UI provides new rank name and rank value inputs.  When documenting a deep classification from `Domain` down to `Species`, the `taxonCoverage.taxonomicClassification` model gets populated correctly with the nested classifications:
```json
{
    "taxonRankName": "Domain",
    "taxonRankValue": "Eukarya",
    "taxonomicClassification": {
        "taxonRankName": "Kingdom",
        "taxonRankValue": "Animalia",
        "taxonomicClassification": {
            "taxonRankName": "Phylum",
            "taxonRankValue": "Chordata",
            "taxonomicClassification": {
                "taxonRankName": "Class",
                "taxonRankValue": "Mammalia",
                "taxonomicClassification": {
                    "taxonRankName": "Order",
                    "taxonRankValue": "Carnivora",
                    "taxonomicClassification": {
                        "taxonRankName": "Family",
                        "taxonRankValue": "Canidae",
                        "taxonomicClassification": {
                            "taxonRankName": "Genus",
                            "taxonRankValue": "Vulpes",
                            "taxonomicClassification": {
                                "taxonRankName": "Species",
                                "taxonRankValue": "Vulpes vulpes"
                            }
                        }
                    }
                }
            }
        }
    }
}
``` 
After submitting the dataset, the resulting landing page view contains a **partial** hierarchy:

<img width="1257" alt="taxa" src="https://user-images.githubusercontent.com/1437565/97369459-96300580-1872-11eb-800f-dccac8fa5ccb.png">

However, the serialized repository version of the metadata only contains every other rank name/value pair:

```xml
<taxonomicCoverage>
    <generalTaxonomicCoverage>Red fox</generalTaxonomicCoverage>
    <taxonomicClassification>
        <taxonRankName>Domain</taxonRankName>
        <taxonRankValue>Eukarya</taxonRankValue>
        <taxonomicClassification>
            <taxonRankName>Phylum</taxonRankName>
            <taxonRankValue>Chordata</taxonRankValue>
            <taxonomicClassification>
                <taxonRankName>Order</taxonRankName>
                <taxonRankValue>Carnivora</taxonRankValue>
                <taxonomicClassification>
                    <taxonRankName>Genus</taxonRankName>
                    <taxonRankValue>Vulpes</taxonRankValue>
                </taxonomicClassification>
            </taxonomicClassification>
        </taxonomicClassification>
    </taxonomicClassification>
</taxonomicCoverage>
```
and the editor only shows the first rank name/value pair:

<img width="1257" alt="taxon-editor" src="https://user-images.githubusercontent.com/1437565/97369863-5e758d80-1873-11eb-8c41-34aef459162b.png">

Oddly enough, the in-memory `MetacatUI.rootDataPackage.models[0].attributes.taxonCoverage[0]` model still contains the full `taxonomicClassification`:

![taxonClasssification](https://user-images.githubusercontent.com/1437565/97370222-2589e880-1874-11eb-8862-bd1a0ffed1a4.png)

When opening the editor a second time, and reloading the EML, we see the metadata loss:

<img width="1257" alt="taxon-editor-2" src="https://user-images.githubusercontent.com/1437565/97370532-caa4c100-1874-11eb-9ef4-20b2f2fc97f0.png">

So it seems like we have 

1. An issue somewhere in `save()` or `serialize()` before sending the EML to the repository
2. An issue in `EMLTaxonView` that clears the `taxonomicClassification` except for the first rank name/value pair.  

**To Reproduce**
1. Upload a new metadata document with a full taxonomic description
2. Look at the `Taxa` section of the editor before navigating away to see the lost metadata
3. Reload the EML into the editor to see the partially restored taxa metadata

**Expected behavior**
The full taxonomic classification should be preserved.

**Desktop (please complete the following information):**
 - OS: MacOS 10.15
 - Browser Chrome
 - Version 86.0.4240.75 (Official Build) (x86_64)

**Comment by mbjones** [comment]
@csjx wouldn't this be related to the recently fixed issue #1523 ? Can you highlight the differences?

**Comment by csjx** [comment]
Ah yes! Jasmine and I had discussed this, and I had it on my todo list to write it up, and just got to it.  I missed that she did also! This is probably just a dupe then. I'll close this one.
